features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/default/templates/item-design/js/api-design.js

```diff
@@ -337,6 +337,28 @@ APIDesigner.prototype.init_controllers = function(){
         API_DESIGNER.render_resource(resource_body);
     });
 
+    this.container.delegate(".delete_parameter", "click", function (event) {
+        console.log("deleting parameter");
+        //var elementToDelete =  $(this).parent().parent();
+        var deleteData = $(this).attr("data-path");
+        var i = $(this).attr("data-index");
+
+        var deleteDataArray = deleteData.split(".");
+        var operations = deleteDataArray[2];
+        var operation = deleteDataArray[3];
+        var paramName = API_DESIGNER.api_doc.paths[operations][operation]['parameters'][i]['name'];
+
+        jagg.message({content: 'Do you want to delete the parameter <strong>' + paramName + '</strong> ?',
```

Comment:
There is a way to use. But we did not implement in that way. Old codes also use the same string messages in jagg.message